diff --git a/CMakeLists.txt b/CMakeLists.txt
index e5d96a3e59..89acc834d5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2655,7 +2655,7 @@ endif()
 
 
 endif()
-if(gRPC_BUILD_TESTS)
+if(FALSE)
 
 add_library(grpc_test_util_unsecure
   test/core/event_engine/test_init.cc
@@ -2730,7 +2730,7 @@ endif()
 
 
 endif()
-
+if(False)
 add_library(grpc_unsecure
   src/core/call/call_arena_allocator.cc
   src/core/call/call_filters.cc
@@ -3248,6 +3248,7 @@ target_link_libraries(grpc_unsecure
 if(_gRPC_PLATFORM_IOS OR _gRPC_PLATFORM_MAC)
   target_link_libraries(grpc_unsecure PUBLIC "-framework CoreFoundation")
 endif()
+endif()
 
 foreach(_hdr
   include/grpc/byte_buffer.h
@@ -3303,7 +3304,7 @@ foreach(_hdr
 endforeach()
 
 
-if(gRPC_INSTALL)
+if(False)
   install(TARGETS grpc_unsecure EXPORT gRPCTargets
     RUNTIME DESTINATION ${gRPC_INSTALL_BINDIR}
     BUNDLE DESTINATION  ${gRPC_INSTALL_BINDIR}
@@ -4228,6 +4229,7 @@ target_link_libraries(grpc++_test_util
 
 endif()
 
+if(FALSE)
 add_library(grpc++_unsecure
   src/cpp/client/call_credentials.cc
   src/cpp/client/channel_cc.cc
@@ -4316,6 +4318,7 @@ target_link_libraries(grpc++_unsecure
   grpc_unsecure
   ${_gRPC_PROTOBUF_LIBRARIES}
 )
+endif()
 
 foreach(_hdr
   include/grpc++/alarm.h
@@ -4536,7 +4539,7 @@ foreach(_hdr
 endforeach()
 
 
-if(gRPC_INSTALL)
+if(False)
   install(TARGETS grpc++_unsecure EXPORT gRPCTargets
     RUNTIME DESTINATION ${gRPC_INSTALL_BINDIR}
     BUNDLE DESTINATION  ${gRPC_INSTALL_BINDIR}
@@ -5521,7 +5524,7 @@ if(_gRPC_PLATFORM_LINUX OR _gRPC_PLATFORM_MAC OR _gRPC_PLATFORM_POSIX)
 
 endif()
 endif()
-if(gRPC_BUILD_TESTS)
+if(FALSE)
 if(_gRPC_PLATFORM_LINUX OR _gRPC_PLATFORM_MAC OR _gRPC_PLATFORM_POSIX)
 
   add_executable(address_sorting_test_unsecure
@@ -6136,7 +6139,7 @@ target_link_libraries(admin_services_end2end_test
 
 
 endif()
-if(gRPC_BUILD_TESTS)
+if(FALSE)
 if(_gRPC_PLATFORM_LINUX OR _gRPC_PLATFORM_MAC OR _gRPC_PLATFORM_POSIX)
 
   add_executable(alarm_test
@@ -15854,7 +15857,7 @@ target_link_libraries(filter_fusion_test
 
 
 endif()
-if(gRPC_BUILD_TESTS)
+if(FALSE)
 
 add_executable(filter_test_test
   ${_gRPC_PROTO_GENS_DIR}/test/core/event_engine/fuzzing_event_engine/fuzzing_event_engine.pb.cc
@@ -24193,7 +24196,7 @@ target_link_libraries(parser_test
 
 
 endif()
-if(gRPC_BUILD_TESTS)
+if(FALSE)
 if(_gRPC_PLATFORM_LINUX OR _gRPC_PLATFORM_MAC OR _gRPC_PLATFORM_POSIX)
 
   add_executable(party_mpsc_test
@@ -24237,7 +24240,7 @@ if(_gRPC_PLATFORM_LINUX OR _gRPC_PLATFORM_MAC OR _gRPC_PLATFORM_POSIX)
 
 endif()
 endif()
-if(gRPC_BUILD_TESTS)
+if(FALSE)
 
 add_executable(party_test
   test/core/promise/party_test.cc
@@ -28654,7 +28657,7 @@ target_link_libraries(server_builder_plugin_test
 
 
 endif()
-if(gRPC_BUILD_TESTS)
+if(FALSE)
 if(_gRPC_PLATFORM_LINUX OR _gRPC_PLATFORM_MAC OR _gRPC_PLATFORM_POSIX)
 
   add_executable(server_builder_test
@@ -28733,7 +28736,7 @@ if(_gRPC_PLATFORM_LINUX OR _gRPC_PLATFORM_MAC OR _gRPC_PLATFORM_POSIX)
 
 endif()
 endif()
-if(gRPC_BUILD_TESTS)
+if(FALSE)
 if(_gRPC_PLATFORM_LINUX OR _gRPC_PLATFORM_MAC OR _gRPC_PLATFORM_POSIX)
 
   add_executable(server_builder_with_socket_mutator_test
@@ -50675,7 +50678,7 @@ generate_pkgconfig(
   "-lgrpc -lupb"
   "-laddress_sorting"
   "grpc.pc")
-
+if(FALSE)
 # grpc_unsecure .pc file
 generate_pkgconfig(
   "gRPC unsecure"
@@ -50686,7 +50689,7 @@ generate_pkgconfig(
   "-lgrpc_unsecure -lupb"
   "-laddress_sorting"
   "grpc_unsecure.pc")
-
+endif()
 # grpc++ .pc file
 generate_pkgconfig(
   "gRPC++"
@@ -50697,7 +50700,7 @@ generate_pkgconfig(
   "-lgrpc++ -lupb"
   "-laddress_sorting"
   "grpc++.pc")
-
+if(FALSE)
 # grpc++_unsecure .pc file
 generate_pkgconfig(
   "gRPC++ unsecure"
@@ -50708,7 +50711,7 @@ generate_pkgconfig(
   "-lgrpc++_unsecure -lupb"
   "-laddress_sorting"
   "grpc++_unsecure.pc")
-
+endif()
 # grpcpp_otel_plugin .pc file
 generate_pkgconfig(
   "gRPC++ OpenTelemetry Plugin"
